weather_stats/weatherstatsapp/weather/src/App.js

```diff
@@ -0,0 +1,15 @@
+import Search from "./Search"
+import Footer from "./Footer"
+
+function App() {
+    return (
+        <div className="bg-gray-50 min-h-screen pt-10">
+            <div className="mx-auto max-w-4xl px-4 sm:px-6 lg:px-8">
+                <Search></Search>
```

Comment:
(Style) Please use shorthand notation on tags when there are no child components `<Search />`, `<Footer />`. Like you never go with `<img src="" alt=""></img>` :)